collections/copernicus-dem.yaml

```diff
@@ -95,22 +95,37 @@ CubeDimensions:
     extent:
       - -180
       - 180
+    reference_system: 4326
   y:
     type: spatial
     axis: y
     extent:
       - -85
       - 85
+    reference_system: 4326
   t:
     type: temporal
     extent:
       - '2010-12-12T05:04:33Z'
       - '2015-01-16T01:37:12Z'
-    step: P5D
   bands:
     type: bands
     values:
       - DEM
       - dataMask
+Summaries:
+  raster:bands:
```

Comment:
No that we use raster:bands here the respective extension should be added to the list of used extensions.

Reply:
Thanks. I think the more correct place is in `_build/stac.hbs` where the entries for other extensions are present as well.

Reply:
Of course, linked the wrong file. Thanks for adding it.